cmake/README.md

```diff
@@ -0,0 +1,53 @@
+# ESP32CommandStation Cmake project
+
+This directory includes the CMake build system as an alternative to platformio.
+See the general setup/usage instructions 
+ https://docs.espressif.com/projects/esp-idf/en/latest/get-started-cmake/index.html
+ 
+ 
+# Setup
+
+```
+git clone --recurse https://github.com/WolfgangGitHub/DCCppESP32.git
+cd DCCppESP32
+git checkout development
```

Comment:
This can be dropped as it will be available on master after merge.

Reply:
right - will do.